src/main/kotlin/racingcar/entity/Position.kt

```diff
@@ -0,0 +1,15 @@
+package racingcar.entity
+
+data class Position(private var _value: Int) {
+    var value: Int = _value
+        get() = _value
+        private set
```

Comment:
```suggestion
        val value: Int get() = _value
```
https://kotlinlang.org/docs/properties.html#backing-properties
가볍게 코틀린 공식문서 읽어보셔도 좋을거같아요!